bench-vortex/src/statpopgen/mod.rs

```diff
@@ -0,0 +1,10 @@
+// SPDX-License-Identifier: Apache-2.0
+// SPDX-FileCopyrightText: Copyright the Vortex contributors
+
```

Comment:
~~How about throwing in a module level comment here along the lines of:~~
```
// Suggested documentation
/// StatPopGen Benchmark
/// 
/// A custom benchmark for statistical and population genetics workloads.
/// Uses real genomics data from gnomAD v3.1.2 HGDP-1kG chromosome 21.
/// 
/// Data source: https://gnomad.broadinstitute.org/
/// Benchmark specification: [No external spec - custom implementation]
/// 
/// Queries test common genomics analytics patterns:
/// - Allele frequency calculations
/// - Hardy-Weinberg equilibrium testing  
/// - Population stratified analysis
/// - Genotype matrix operations
```

Reply:
Done (though I wrote a fresh doc myself)